src/core_processor/camera_calibration/camera_calibration.py

```diff
@@ -0,0 +1,22 @@
+from src.cameras.capture.frame_payload import FramePayload
+from src.core_processor.board_detection.board_detection import BoardDetection
+
+
+class CameraCalibration:
+    def __init__(self, board_detection: BoardDetection):
+        self._bd = board_detection
+
+    def perform_calibration(self):
+        self._bd.process(
+            show_window=True,
+            post_processed_frame_cb=self._calibration_work,
+        )
+
+    def _calibration_work(self, frame: FramePayload):
+        print("found frame", frame.timestamp)
```

Comment:
consider this method your way of receiving a processed frame (success, image, timestamp tuple). The work necessary to implement Cam calibration can go here.